tess_conf_filter.py

```diff
@@ -0,0 +1,84 @@
+"""
+This is a series of functions that take in a jpg or pdf, runs it through tessaract
+ocr, extracting words and confidence levels per word, then filters out words
+below a specified confidence level.
+"""
+
+from PIL import Image
+import pytesseract
+import pdf2image
+import time
+import re
+import pandas as pd
```

Comment:
I forgot to mention in the description: this uses pandas, which is not currently in the requirements.txt.